lib/models/notices.js

```diff
@@ -124,6 +171,21 @@ module.exports.errors = errors;
         this.message = 'type as attribute ' + msg;
         this.httpCode = 400;
     };
+    errors.InvalidLocation = function InvalidLocation(msg) {
```

Comment:
Do we have the list of errors described in some point of the documentation? In that case, the three new ones included in this PR should be added.